docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
I discussed this briefly with @himanshug this morning. While we could move the schemaMappings section to DimensionsSpec and just reference the mapping in the the metricsSpec, we don't particularly like the approach for several reasons:

1. It complicates the DimensionsSpec in a way not needed by other input parsers. 
1. There would still be an inconsistency between how metrics and dimensions are specified
1. This approach doesn't lend itself to prevalidating the schema mappings the same way the current approach does

I think what you are proposing could be made to work, but would be more of an invasive change and require more complicated code to support. If others agree, I can take a look at making the change, but my preference is to leave the code the way it is.